web/event_handler.go

```diff
@@ -20,20 +20,22 @@ type event struct {
 }
 
 type eventHandler struct {
-	id              int
-	serviceRegistry service.ServiceRegistry
-	marathon        marathon.Marathoner
-	eventQueue      <-chan event
+	id                 int
+	serviceRegistry    service.ServiceRegistry
+	marathon           marathon.Marathoner
+	eventQueue         <-chan event
+	eventQueueCapacity int
```

Comment:
Do we really need this value? Can't just use [cap](https://golang.org/pkg/builtin/#cap)